installing/installing_aws/installing-aws-account.adoc

```diff
@@ -21,7 +21,7 @@ include::modules/installation-aws-regions.adoc[leveloffset=+1]
 == Next steps
 
 * Install an {product-title} cluster:
-** xref:../../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[Quickly install a cluster] with default options
-** xref:../../installing/installing_aws/installing-aws-customizations.adoc#installing-aws-customizations[Install a cluster with cloud customizations]
-** xref:../../installing/installing_aws/installing-aws-network-customizations.adoc#installing-aws-network-customizations[Install a cluster with network customizations]
-** xref:../../installing/installing_aws/installing-aws-user-infra.adoc#installing-aws-user-infra[Install a cluster on infrastructure that you provision]
+** xref:../../installing/installing_aws/installing-aws-default.adoc#installing-aws-default[Quickly install a cluster] with default options (Installer Provisioned Infrastructure)
```

Comment:
I'd change them all to "on installer-provisioned infrastructure" because the IBM style guide says:

> Avoid using parentheses in running text because text in parentheses is decreased in importance.

Reply:
Thanks - I'll make the changes!